roles/container-engine/docker/tasks/main.yml

```diff
@@ -189,24 +135,8 @@
   retries: 4
   delay: "{{ retry_stagger | d(3) }}"
   notify: restart docker
-  ignore_errors: true
   when: ansible_os_family in ["ClearLinux"]
 
-- name: get available packages on Ubuntu
-  command: apt-cache policy docker-ce
-  when:
-    - docker_task_result is failed
-    - ansible_distribution == 'Ubuntu'
-  register: available_packages
-  check_mode: false
```

Comment:
ditto: Why can we remove this task?

Reply:
This is a debug task for ubuntu only if install fails, this is useless 99.99% of the time

Reply:
After reading this PR again, I guess the reason we can remove here is this PR removes line 192 `ignore_errors: true` to make the ansible-playbook run stop immediately then this debugging task become useless(never run).